caffe2/operators/mean_op.h

```diff
@@ -20,35 +21,45 @@ class MeanOp final : public Operator<Context> {
   template <typename T>
   bool DoRunWithType() {
     auto& input0 = Input(0);
-
     auto* output = Output(0, input0.sizes(), at::dtype<T>());
     output->CopyFrom(input0, true /*async*/);
```

Comment:
Here when InputSize == 2, that will not be very efficient since we have to copy and then add. Actually InputSize == 2 will be one of the most common usecases. So I'd suggest optimize it.